frontend/packages/ui/src/Button.tsx

```diff
@@ -1,200 +1,35 @@
 import React, { ButtonHTMLAttributes, FunctionComponent } from 'react'
-import styled from 'styled-components'
-
 import { getActiveComponent } from './ButtonVariants'
-import { Icon, IconNamesType } from './Icon'
-import { Loader } from './Loader'
-import { Fonts } from './styles/fonts'
-
-/**
- * Possible values for `iconsPosition` prop of `Button` component.
- * Icon alignment relative to button label
- */
-export const IconsPosition = ['left', 'right', 'both'] as const
-export type IconsPosition = typeof IconsPosition[number]
 
-/**
- * Possible values for `variant` prop of `Button` component.
- */
-export const ButtonVariant = [
-  'full',
-  'outline',
-  'text',
-  'badge',
-  'icon',
-] as const
-export type ButtonVariant = typeof ButtonVariant[number]
+export const ButtonAppearance = ['system', 'ghost', 'ghostSmall'] as const
```

Comment:
Reference some ui library f.e https://mui.com/components/buttons/
We have color, but don't use green, orange, etc. use as they did 'success', 'error' etc..
Tomorrow we can change success to be a blue color without changing the usage of the Button.

Avoid camel case in a string like 'ghostSmall', we are acting as a ui-library just like material-ui and you won't see there ghostSmall, but you can see ghost as appearance and small as a size.

@mstojanovicbloxico I'm adding you so that you know as well what we are aiming.

Reply:
Done